lib/model/rwfolder.go

```diff
@@ -789,21 +785,8 @@ func (f *sendReceiveFolder) deleteDir(file protocol.FileInfo, matcher *ignore.Ma
 		return
 	}
 
-	// Delete any temporary files lying around in the directory
-	dir, _ := os.Open(realName)
-	if dir != nil {
-		files, _ := dir.Readdirnames(-1)
-		for _, dirFile := range files {
-			fullDirFile := filepath.Join(file.Name, dirFile)
-			if ignore.IsTemporary(dirFile) || (matcher != nil &&
-				matcher.Match(fullDirFile).IsDeletable()) {
-				os.RemoveAll(filepath.Join(f.dir, fullDirFile))
-			}
-		}
-		dir.Close()
-	}
+	err = f.folder.deleteDir(f.dir, file, matcher)
```

Comment:
I didn't want to emit an event that matches the "normal" folder sync operations.
Within validateAndUpdateLocalChanges it already emits an event "LocalChangeRejected".